src/components/Modals/ReceiveManaModal/ReceiveManaModal.tsx

```diff
@@ -28,7 +28,7 @@ const ReceiveManaModal = ({ name, onClose, metadata }: ModalProps) => {
       <Modal.Content>
         <div className="message"> {t('receive_mana_modal.message')} </div>
         <Field
-          value={`${walletAddress.slice(0, 6)}...${walletAddress.slice(-4)}`}
+          value={`${address.slice(0, 6)}...${address.slice(-4)}`}
```

Comment:
have we something similar at `decentraland-dapps`?

Reply:
there was a helper in the marketplace for this but not in dapps

Reply:
should we need to create an issue to move it to dapps (both places)?

Reply:
sure, let's add an issue to decentraland-dapps and i ll add it for the next release